src/entities/room.js

```diff
@@ -0,0 +1,29 @@
+const BaseEntity = require('./base')
+
+class Room extends BaseEntity {
+    set title(title) {
```

Comment:
Is `set` used only to change existing properties or can it also add new ones to the instance of the class?

Reply:
You can do anything in set methods.
It is the way to define `room.title = 'New title'` method. 'New title' is an argument. It has some restrictions as i know: 1. it can't return a result, and it can't be `async`. You can do the rest inside.